src/guides/v2.4/release-notes/commerce-2-4-2.md

```diff
@@ -2033,18 +2303,22 @@ We have fixed hundreds of issues in the Magento 2.4.2 core code.
 
 <!-- ENGCOM-8062 -->
 
-*  Magento now displays selected configurable product attribute options in the wishlist page as expected.  [GitHub-24091](https://github.com/magento/magento2/issues/24091),  [GitHub-22503](https://github.com/magento/magento2/issues/22503)
+*  Magento now displays selected configurable product attribute options in the wishlist page as expected. [GitHub-24091](https://github.com/magento/magento2/issues/24091),  [GitHub-22503](https://github.com/magento/magento2/issues/22503)
 
 <!-- ENGCOM-7779 -->
 
-*  Implemented `ActionInterface` for \Magento\Wishlist\Controller\Shared\Allcart. [GitHub-29537](https://github.com/magento/magento2/issues/29537)
+*  Implemented `ActionInterface` for `\Magento\Wishlist\Controller\Shared\Allcart`. [GitHub-29537](https://github.com/magento/magento2/issues/29537)
 
 ## Known issues
 
-**Issue**: The `deleteRequisitionListOutput` query returns details about the deleted requisition list instead of the remaining requisition lists. <!--- MC-39894-->
-
 **Issue**: Merchants must activate the Fastly Force TLS functionality from the Admin to enable the global HTTP to HTTPS redirect for all store pages. See the [Redirect HTTP to HTTPS for all pages on Cloud (Force TLS)](https://support.magento.com/hc/en-us/articles/360006296953) Knowledge Base article. <!--- MC-39988-->
 
+**Issue**: You cannot use the `addConfigurableProductToCart` GraphQL mutation to add a configurable product to a cart in a non-default store view in a multi-store deployment that runs Inventory. (Luma storefronts are not affected.) Magento displays this error: `Could not add item to cart. Please check required options and try again`. **Workaround**: None. [GitHub-31660](https://github.com/magento/magento2/issues/31660)<!--- PWA-1298-->
```

Comment:
Are you sure there's no workaround? Last I heard, using `addProductsToCart` would work.